public/plugin.tsx

```diff
@@ -136,6 +146,11 @@ export class AlertingPlugin implements Plugin<void, AlertingStart, AlertingSetup
         DEFAULT_NAV_GROUPS.observability,
         navLinks
       );
+
+      core.chrome.navGroup.addNavLinksToGroup(
+        DEFAULT_NAV_GROUPS['security-analytics'],
+        navLinks
+      );
```

Comment:
Alerting is not supposed to be added to security-analytics, can we raise a PR to revert?